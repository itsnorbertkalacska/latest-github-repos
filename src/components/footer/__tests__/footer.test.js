import { render } from "@testing-library/react";

import { Footer } from "../index";

describe("Footer", () => {
  it("renders component properly", () => {
    const { container } = render(<Footer />);
    // eslint-disable-next-line testing-library/no-node-access
    expect(container.firstChild).toMatchSnapshot();
  });
});
