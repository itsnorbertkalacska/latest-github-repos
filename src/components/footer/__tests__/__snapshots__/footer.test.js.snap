// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders component properly 1`] = `
<footer
  class="sc-bczRLJ iAVRiY"
>
  <p>
    2022
     Â© 
    Discover Trending GitHub Repos
  </p>
</footer>
`;
